content/docs/command-reference/exp/show.md

```diff
@@ -74,7 +74,12 @@ will be generated using the same data from the table.
 - `-A`, `--all-commits` - include experiments derived from all Git commits, as
   well as from the last one. This prints all experiments in the project.
 
-- `-n <num>`, `--num <num>` - show the last `num` commits from HEAD.
+- `--rev <commit>` - show experiments derived from the specified `<commit>` as
+  baseline. Defaults to `HEAD` if none of `--rev`, `-a`, `-T`, `-A` is used.
+
+- `-n <num>`, `--num <num>` - show experiments from the `--rev` baseline and
+  from `-num` commits before it. `num` accepts negative values (similar to
+  `git log -n`).
```

Comment:
```suggestion
- `-n <num>`, `--num <num>` - show experiments from the previous `num`
  commits (before the `--rev` baseline). For example `-n -2` includes
  experiments from `HEAD` and its 2 first ancestors.
```

Reply:
Hmmm wait it looks like I was confused in here. We should revert... I'll open another suggestion.